src/config/config.go

```diff
@@ -74,10 +85,19 @@ type BtcScanner struct {
 	ConfirmationsRequired int64         `mapstructure:"confirmations_required"`
 }
 
+// EthScanner config for ETH scanner
+type EthScanner struct {
+	// How often to try to scan for blocks
+	ScanPeriod            time.Duration `mapstructure:"scan_period"`
+	InitialScanHeight     int64         `mapstructure:"initial_scan_height"`
+	ConfirmationsRequired int64         `mapstructure:"confirmations_required"`
+}
+
 // SkyExchanger config for skycoin sender
 type SkyExchanger struct {
 	// SKY/BTC exchange rate. Can be an int, float or rational fraction string
 	SkyBtcExchangeRate string `mapstructure:"sky_btc_exchange_rate"`
+	SkyEthExchangeRate string `mapstructure:"sky_eth_exchange_rate"`
```

Comment:
Validate this field in `Config.Validate()`

Reply:
done